migrations/lib/database/schema/loader.rb

```diff
@@ -102,6 +102,16 @@ def datatype_for(column)
       end
     end
 
+    def nullable_for(column, config)
+      modified_column = config.dig(:columns, :modify)&.find { |col| col[:name] == column.name }
+      return modified_column[:nullable] unless modified_column&.dig(:nullable).nil?
```

Comment:
It could be simplified

```suggestion
      return modified_column[:nullable] if modified_column&.key?(:nullable)
```